Suppress test failure on buildds which don't have /etc/hosts installed.

diff -Naur --exclude=debian perl-5.10.0.orig/lib/Net/hostent.t perl-5.10.0/lib/Net/hostent.t
--- perl-5.10.0.orig/lib/Net/hostent.t	2007-11-18 02:26:24.000000000 +1100
+++ perl-5.10.0/lib/Net/hostent.t	2007-11-26 06:07:15.000000000 +1100
@@ -16,6 +16,9 @@
     }
     if ($^O eq 'MacOS' || ($^O eq 'irix' && $Config{osvers} == 5)) {
 	plan skip_all => "Test relies on resolution of localhost, fails on $^O ($Config{osvers})";
+    } elsif (!-f '/etc/hosts') {
+	# /etc/hosts is not always present on buildds
+	plan skip_all => "Test requires /etc/hosts: not found";
     }
 }
 
